jep/1/README.adoc

```diff
@@ -467,26 +487,64 @@ JEP will occur.
 [[accepted]]
 ==== Accepting a JEP
 
-For a JEP to be "Accepted" it must meet certain minimum criteria:
-
-* It must be a clear and complete description of the proposed enhancement.
-* The enhancement must represent a net improvement.
-* The proposed implementation, if applicable, must be solid, must not
-  complicate Jenkins unduly, and must be the same license as the component the
-  proposal is meant to added to (or MIT licensed by default).
-
-Once a JEP has been accepted, the implementation must be completed.
+For a JEP to be "Accepted" it must meet certain minimum criteria.
+It must:
+
+* provide a net improvement.
+* represent the consensus of the community,
+  including documentation of dissenting opions.
+* clearly define the scope and features of the proposed enhancement.
+* describe a completed design that addesses any major design questions.
+
+For JEPs that include implementation based on the proposal,
+the implementation must meet certain minimum criteria.
+It must:
+
+* be solid and have progressed enough to resolve major design or scope questions.
+* not complicate Jenkins unduly.
+* have the same license as the component the
+  proposal is meant to be added to (or MIT licensed by default).
+
+By marking a JEP as "Accepted" the Reviewer indicates they believe that the JEP has
+clear scope, design completeness, community consensus, and in-progress implementation.
+Without all of these a JEP cannot be accepted.
+For this reason, it is not unusual for JEPs to remain in "Draft" state
+even after they have strong community support and progressing implementation.
+They must still pass the other criteria, such as scoping and design completeness.
+
+Once a JEP has been accepted, the implementation must continue to progress
+and eventually be completed.
 The Jenkins project values contribution over "talk"
 footnote:[https://jenkins.io/project/governance/#meritocracy], and as such the
 implementation is of utmost importance to moving any proposal (Standards or
 Process) forward.
 
+Ideally, all changes to a JEP should be completed before it is "Accepted",
+but surprises may still occur.  Changes might be minor changes,
+such as clarifications or typo fixes, or major changes,
+which would alter the intent, scope, API, or core behavior of the JEP.
+
+All changes to an already "Accepted" JEP, must be submitted via PR as usual.
+In the case of major changes,
```

Comment:
I would also expect JEPs to have versioning and a revision history section for such case

Reply:
Part of the point of this change is to avoid the need for a "Revisions/Versioning" section.  

Since the JEP isn't final, it would still be the same version.  

The commit history could be considered sufficient revision history tracking, right?  
The discussion thread link ( https://github.com/jenkinsci/jep/pull/76/files#diff-43894e615b82e2536b1341c33ff49afeR532 ) would add context to any revisions. 

Changes that would require a new "version" would be a new JEP, started after the current JEP is final.